src/Analyzers/CSharp/CodeFixes/ConvertNamespace/ConvertNamespaceTransform.cs

```diff
@@ -66,51 +168,47 @@ public static BaseNamespaceDeclarationSyntax Convert(BaseNamespaceDeclarationSyn
 
         private static FileScopedNamespaceDeclarationSyntax ConvertNamespaceDeclaration(NamespaceDeclarationSyntax namespaceDeclaration)
         {
-            // We move leading and trailing trivia on the open brace to just be trailing trivia on the semicolon, so we preserve
```

Comment:
there was a lot of wonky logic here moving trivia around and adding elasticity.  THis has been changed to be much simpler, with no elastic trivia anymore.